Libraries/CMSIS/Device/Maxim/MAX78000/Include/bbfc_regs.h

```diff
@@ -2,10 +2,11 @@
  * @file    bbfc_regs.h
  * @brief   The BBFC block has been renamed to GCFR.  
  *          This include file is for backwards compatibility only.
+ * @note    This file is @deprecated.
```

Comment:
Adding this note as a unique token to exclude checking these registers files for the new Register_SVD workflow. Let me know if you think there's a better way to handle this.